atlas

```diff
@@ -4,8 +4,76 @@
 
 # @getoptions
 parser_definition() {
-	setup       REST help:usage -- \
-				"Atlas is a CLI tool that has essentially one command: \`atlas pull\`\n \nAtlas defaults to using a configuration file named \`atlas.yml\` placed\nin the root directory. Configuration file:\n \npull:\n  branch: <branch-name>\n  directory <directory-name>\n  repository: <organization-name>/<repository-name>\n \nAtlas can also use a configuration file in a different path using the \`--config\` flag\nafter \`atlas\`: \`atlas pull --config config.yml\`.\n \nAtlas can also be used without a configuration file by using the flags below after\n\`atlas pull\`.\n \n\`-b\` or \`--branch\`\n\`-r\` or \`--repository\`\n\`-d\` or \`--directory\`" ''
+    _ATLAS_USAGE_HELP="Atlas is a CLI tool that has essentially one command: \`atlas pull\`
+
+Configuration file:
+
+    Atlas defaults to using a configuration file named \`atlas.yml\` placed
+    in the root directory. Configuration file:
+
+    pull:
+      repository: <organization-name>/<repository-name>
+      branch: <branch-name>
+      directory: <repo-directory-name>:<local-dir-name>
+      filter: ar de fr_CA
```

Comment:
The rest of the configuration file documentation here uses descriptions instead of examples. Maybe something like
```suggestion
      filter: <lang-code>
```
would work? The only issue I have with using `<lang-code>` is that the filter isn't technically language specific. I'm very open to other ideas for what to put here.

Reply:
Makes sense. Lang-code isn't what I prefer, but it's 99% of the time it'll just be that.

I added `<pattern>`. Please see the latest [force-push](https://github.com/openedx/openedx-atlas/compare/10f9d0e96e3ad9ad3b74d0359379522ddd69c9b1..946b501f239fa2de041932153118bff931a1da3c) update for the rest of the changes.